grails-app/domain/org/pih/warehouse/product/Product.groovy

```diff
@@ -589,12 +600,12 @@ class Product implements Comparable, Serializable {
         def results = ProductCatalogItem.executeQuery("select pci.productCatalog.color " +
```

Comment:
Do we still need getColor() ?

Reply:
@DominikaHatala It is still used. 
![Selection_446](https://user-images.githubusercontent.com/12113312/101789761-f5379a00-3b01-11eb-9cf3-ef03c9277267.png)